src/projection/animation/mix-values.ts

```diff
@@ -51,13 +54,22 @@ export function mixValues(
         followRadius ||= 0
         leadRadius ||= 0
 
-        target[borderLabel] = Math.max(
-            mix(asNumber(followRadius), asNumber(leadRadius), progress),
-            0
-        )
+        const canMix =
+            followRadius === 0 ||
+            leadRadius === 0 ||
+            isPx(followRadius) === isPx(leadRadius)
```

Comment:
I'm confused why we're not adding `percent.test(leadRadius) === percent.test(followRadius)` to this?

Reply:
But the test case seems to be still passing...

Reply:
This is essentially testing if they're different

Reply:
Ah right! So in the purely hypothetical case that there would be a third format, we would be fucked.
Let's add a comment about that then, because I think it's pretty confusing.